### PR TITLE
TRAIN-837 integrations

### PR DESCRIPTION
**Pull request by stevedatadog**
This is a new assets directory for the Integrations course. After the updated course goes live, I will submit another PR to delete the old assets dir, integrations-intro-od.

### CODE REVIEW

File: courses/intro-integrations/coder.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "query": {
+      "folder": "/root/lab"
+    },
+    "update": {
+      "checked": 1665349162360,
+      "version": "4.7.1"
+    }
```
Comment (arosenkranz) [inline courses/intro-integrations/coder.json:8]: Does this lock in the version so there's no notification in the IDE?

File: courses/intro-integrations/new_conftest.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    # 1. Spins up the services defined in the compose file
+    # 2. Waits for the url to be available before running the tests
+    # 3. Tears down the services when the tests are finished
+    with docker_run(compose_file, endpoints=[URL]):
+        yield INSTANCE
+
+
+@pytest.fixture
+def instance():
+    return INSTANCE.copy()
```
Comment (arosenkranz) [inline courses/intro-integrations/new_conftest.py:27]: Is a return needed?

**Review by arosenkranz — APPROVED** [review]
Not sure if it matters, but it seems like all of the files don't have returns at the end.

File: courses/intro-integrations/coder.json (hunk 1 above)
Comment (stevedatadog) [reply courses/intro-integrations/coder.json:8]: Yes. I don't know how long that timestamp is good for, though. It's part of new VM images, so if we ever see a vs code update notification, we'll know. (might be able to set it far in the future, or to "today" on every startup.)

**Review by arosenkranz — APPROVED** [review]

**Pull request merged**